src/flake8/options/config.py

```diff
@@ -60,6 +61,10 @@ def load_config(
 
     cfg = configparser.RawConfigParser()
     if config is not None:
+        if not os.path.isfile(config):
+            raise exceptions.ExecutionError(
+                "The specified config file does not exist"
```

Comment:
Why wouldn't you include the config value in the exception?

Reply:
have added this now